### PR TITLE
BigQuery _EnumProperty ValueError messages are not displayed properly

### PR DESCRIPTION
**Pull request by smasue**
I found this very small bug in the error message, so I fixed it!

### CODE REVIEW

**Comment by dhermes** [comment]
LGTM. Thanks!

Can you let us know once you've signed the CLA (unless the bot makes an automated reply)?

**Comment by smasue** [comment]
I've just signed the CLA

File: bigquery/google/cloud/bigquery/_helpers.py [hunk 1]
```diff
@@ -318,7 +318,7 @@ def _validate(self, value):
         :raises: ValueError if value is not allowed.
         """
         if value not in self.ALLOWED:
-            raise ValueError('Pass one of: %s' ', '.join(self.ALLOWED))
```
Comment (dhermes) [inline bigquery/google/cloud/bigquery/_helpers.py]: No worries here, but a point of note when using percent (`%`) formatting in Python. To be extra super-duper safe, you should always pass a tuple:

```python
>>> format = 'see %s'
>>> value = 'this'
>>> format % value
'see this'
>>> value = ('a', 'real', 'tuple', 9)
>>> format % value
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: not all arguments converted during string formatting
>>> format % (value,)
"see ('a', 'real', 'tuple', 9)"
```

In this case it's obvious we aren't accidentally passing a tuple as the single argument

Comment (smasue) [inline bigquery/google/cloud/bigquery/_helpers.py]: Yes you are right! Do you want me to do another PR for it?

Comment (dhermes) [inline bigquery/google/cloud/bigquery/_helpers.py]: Naw no worries, thanks for the contribution!
